packages/core/src/lib/__tests__/modal.spec.ts

```diff
@@ -1,38 +1,59 @@
 import { describe, it, expect, vi } from 'vitest';
-import { render, fireEvent } from '@testing-library/svelte';
+import { render, fireEvent, screen } from '@testing-library/svelte';
 import Modal from '../modal.svelte';
 
 describe('Modal', () => {
   it('should emit close event when close icon button is clicked', async () => {
-    const { getByTitle, component } = render(Modal, {
+    const { component } = render(Modal, {
       open: true,
     });
 
-    const closeButton = getByTitle('Close modal');
+    const closeButton = screen.getByTitle('Close modal');
```

Comment:
[Not new code, improvement opportunity] `getByTitle` is not particularly accessible. It think this should be:

```suggestion
    const modal = screen.getByRole('dialog')
    const closeButton = within(modal).getByRole('button', { name: /close/iu });
```

You'll need to `import { ..., within } from '@testing-library/svelte'` for this to work